packages/hashi/src/iframeClient.js

```diff
@@ -81,21 +81,14 @@ export default class SandboxEnvironment {
         this.H5P.iframeInitialize(this.iframe.contentWindow);
         this.xAPI.iframeInitialize(this.iframe.contentWindow);
         patchIndexedDB(this.contentNamespace, this.iframe.contentWindow);
-        this.mutationObserver.observe(this.iframe.contentDocument, {
-          attributes: true,
-          childList: true,
-          subtree: true,
-        });
       } catch (e) {
+        console.debug(e);
```

Comment:
Is this committed intentionally?

Reply:
Yes - the reason one of the bugs I fixed in here went unnoticed for so long was because the error was silently swallowed.